e2e/docgen/lib/numbers.js

```diff
@@ -0,0 +1,71 @@
+const { ethers } = require('ethers');
+const numbro = require('numbro');
+
+function rawValue(value) {
+  if (value instanceof ethers.BigNumber) {
+    if (ethers.constants.MaxUint256.eq(value)) {
+      return `<code>MaxUint256</code> / <code>${value.toHexString()}</code>`;
+    }
+    if (ethers.constants.MaxInt256.eq(value)) {
+      return `<code>MaxInt256</code> / <code>${value.toHexString()}</code>`;
+    }
+    return `<code>${value.toString()}</code> / <code>${value.toHexString()}</code>`;
+  }
+  return `<code>${JSON.stringify(value)}</code>`;
+}
+
+function readableBigWei(value) {
+  if (!(value instanceof ethers.BigNumber)) {
+    return rawValue(value);
+  }
+  if (ethers.constants.MaxUint256.eq(value)) {
+    return '<code>MaxUint256</code>';
+  }
+  if (ethers.constants.MaxInt256.eq(value)) {
```

Comment:
Do be totally correct here I think this is slighlyt better:
`ethers.constants.MaxInt256.eq(value.abs())`
as value could be neg

Reply:
good one, yep will add this fix. Realistically we probably never ever use it. But worth adding anyway